railties/lib/rails/test_help.rb

```diff
@@ -14,10 +14,12 @@
 if defined?(ActiveRecord::Base)
   ActiveRecord::Migration.maintain_test_schema!
 
-  class ActiveSupport::TestCase
-    include ActiveRecord::TestFixtures
-    self.fixture_path = "#{Rails.root}/test/fixtures/"
-    self.file_fixture_path = fixture_path + "files"
+  module ActiveSupport
+    class TestCase
```

Comment:
Yes, absolutely :/ It will need further investigation, I will try to do it ASAP or report it upstream.